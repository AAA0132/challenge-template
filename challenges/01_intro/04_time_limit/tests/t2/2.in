569473682507321219892
