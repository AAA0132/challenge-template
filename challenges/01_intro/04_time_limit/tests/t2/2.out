162150137534224645177601029261607168855778
